identity-service/src/app.js

```diff
@@ -112,6 +105,13 @@ class App {
           )
           cluster.fork(worker.process.env)
         })
+
+        const audiusInstance = await this.configureAudiusInstance()
+        cluster.fork({ WORKER_TYPE: 'notifications' })
+
+        await this.configureRewardsAttester(audiusInstance)
+        await this.configureDiscoveryNodeRegistration(audiusInstance)
+        await this.configureReporter()
```

Comment:
this is the part I just wanted to confirm with @jowlee that it's okay to move to after spawning child processes. there's no communication from the workers that relies on the master having first run these functions right?

Reply:
you should be good to make this change 👍